x-pack/plugins/security_solution/public/management/pages/policy/view/policy_settings_form/components/reputation_service.tsx

```diff
@@ -0,0 +1,105 @@
+import type { PolicyFormComponentCommonProps } from '../types';
+import { ProtectionModes } from '../../../../../../../common/endpoint/types';
+
+interface ReputationServiceProps extends PolicyFormComponentCommonProps {
+  protection: PolicyProtection;
+}
+
+export const ReputationService = React.memo(
+  ({
+    policy,
+    onChange,
+    mode,
+    protection,
+    'data-test-subj': dataTestSubj,
+  }: ReputationServiceProps) => {
+    const isEditMode = mode === 'edit';
+
+    const getTestId = useTestIdGenerator(dataTestSubj);
+
+    const protectionTurnedOn = policy.windows.behavior_protection.mode !== ProtectionModes.off;
+
+    const checkboxChecked =
+      policy.windows.behavior_protection.reputation_service && protectionTurnedOn;
+
+    const checkboxDisabled = !isEditMode || !protectionTurnedOn;
+
+    const handleChange = useCallback(
+      (event) => {
+        const newPayload = cloneDeep(policy);
+        newPayload.windows.behavior_protection.reputation_service = event.target.checked;
+        newPayload.mac.behavior_protection.reputation_service = event.target.checked;
+        newPayload.linux.behavior_protection.reputation_service = event.target.checked;
+
+        onChange({ isValid: true, updatedPolicy: newPayload });
+      },
+      [policy, onChange]
+    );
+    return (
+      <div data-test-subj={getTestId()}>
+        <EuiSpacer size="m" />
+        <SettingCardHeader>
+          <EuiFlexGroup gutterSize="xs">
+            <EuiFlexItem grow={false} data-test-subj={getTestId('label')}>
+              <EuiText size="s">
+                <h4>
+                  <FormattedMessage
+                    id="xpack.securitySolution.endpoint.policyDetailsConfig.reputationService"
+                    defaultMessage="Reputation service"
+                  />
+                </h4>
+              </EuiText>
+            </EuiFlexItem>
+            <EuiFlexItem grow={false} data-test-subj={getTestId('tooltipIcon')}>
+              <EuiIconTip
+                position="right"
+                data-test-subj={getTestId('tooltip')}
+                content={
+                  <FormattedMessage
+                    id="xpack.securitySolution.endpoint.policyDetailsConfig.reputationServiceTooltip"
+                    defaultMessage="This option enables/disables the Reputation Service feature in Endpoint. When the option is ON, Endpoint will reach out to a Cloud API for additional malware analysis. When it's OFF, Endpoint will not reach out to the Cloud API." // TODO: update message
+                  />
+                }
+              />
+            </EuiFlexItem>
+          </EuiFlexGroup>
+        </SettingCardHeader>
+        <EuiSpacer size="s" />
+        <EuiCheckbox
+          data-test-subj={getTestId('checkbox')}
+          id={`${protection}ReputationServiceCheckbox}`}
+          onChange={handleChange}
+          checked={checkboxChecked}
+          disabled={checkboxDisabled}
```

Comment:
I don' think you need this prop `disabled`.

Also - you are missing code to display this option in `mode: "view"`.

_FYI: to test in view mode, load up the test user/roles and login as `hunter` user._

Reply:
I need it for when whole card is turned off, otherwise it looks like this. Same is done with other fields. 
![Screenshot 2023-07-13 at 17 21 50](https://github.com/elastic/kibana/assets/29123534/3e5b631a-dfa5-4177-9070-9ec1ddcd56d2)

Added view part.